devtool/client/src/components/DevToolsPanel.tsx

```
@@ -0,0 +1,115 @@
+                        <button
+                            key={key}
+                            className="devtools-btn devtools-btn-primary"
+                            onClick={() => handleFeatureSelect(key)}
+                            style={{
+                                display: 'flex',
+                                alignItems: 'center',
+                                gap: '8px',
+                                justifyContent: 'flex-start',
+                                padding: '12px 16px'
+                            }}
+                        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)